k8s/namespaces/ia/ia-hearings-api/ia-hearings-api.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-hearings-api
+spec:
+  releaseName: ia-hearings-api
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/ia-hearings-api
+  values:
+    java:
+      disableTraefikTls: true
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 5
+        targetCPUUtilizationPercentage: 80
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/hearings-api:latest
```

Comment:
Production should be as example  `prod-4f6aa00-20220718112253`
- ImagePolicy also needs added 

Example
`image: hmctspublic.azurecr.io/ia/hearings-api:prod-4f6aa00-20220718112253 #{"$imagepolicy": "flux-system:ia-hearings-api"}`

Reply:
Thanks Thomas, added those now.